homeassistant/components/samsungtv/config_flow.py

```diff
@@ -78,15 +77,77 @@ def _get_entry(self):
             data=data,
         )
 
+    async def _async_set_device_unique_id(self, raise_on_progress=True):
+        """Set device unique_id."""
+
+        if self._id:
+            await self.async_set_unique_id(
+                self._id, raise_on_progress=raise_on_progress
+            )
+            self._abort_if_unique_id_configured()
```

Comment:
```suggestion
```

We should drop this since the user could switch devices in `async_step_user`

Reply:
Done